docker/genericsql/verify.py

```diff
@@ -1,15 +1,14 @@
 import sqlalchemy
-import pymysql
```

Comment:
Why did you remove this? We use this to make sure the library is imported correctly